src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -150,8 +151,15 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
   const mixpanel = getMixPanel()
   const greenColor = useColorModeValue('green.600', 'green.200')
   const dispatch = useAppDispatch()
+<<<<<<< HEAD
```

Comment:
ser

Reply:
dumb mergefix, fixed in [b0899b1](https://github.com/shapeshift/web/pull/7270/commits/b0899b12fd4e64a9c180e41ac55591320022936b)